packages/cli/src/__tests__/commands/Studio.test.ts

```diff
@@ -48,7 +48,7 @@ describe('studio', () => {
   })
 
   afterAll(async () => {
-    await studio.instance!.stop()
+    studio.instance!.stop()
```

Comment:
A bit surprised by the removal of the await 🤔 

Is that intended?

Reply:
Yeah, the function is not an async promise so the `await` didn't have any effect.